src/icalendar/tests/test_fixed_issues.py

```diff
@@ -42,47 +42,3 @@ def test_issue_116(self):
             event.to_ical(),
             icalendar.Event.from_ical(event.to_ical()).to_ical()
         )
-
-    def test_issue_237(self):
-        """Issue #237 - Fail to parse timezone with non-ascii TZID"""
-
-        ical_str = ['BEGIN:VCALENDAR',
-                    'BEGIN:VTIMEZONE',
-                    'TZID:(UTC-03:00) Brasília',
-                    'BEGIN:STANDARD',
-                    'TZNAME:Brasília standard',
-                    'DTSTART:16010101T235959',
-                    'TZOFFSETFROM:-0200',
-                    'TZOFFSETTO:-0300',
-                    'RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=3SA;BYMONTH=2',
-                    'END:STANDARD',
-                    'BEGIN:DAYLIGHT',
-                    'TZNAME:Brasília daylight',
-                    'DTSTART:16010101T235959',
-                    'TZOFFSETFROM:-0300',
-                    'TZOFFSETTO:-0200',
-                    'RRULE:FREQ=YEARLY;INTERVAL=1;BYDAY=2SA;BYMONTH=10',
-                    'END:DAYLIGHT',
-                    'END:VTIMEZONE',
-                    'BEGIN:VEVENT',
-                    'DTSTART;TZID=\"(UTC-03:00) Brasília\":20170511T133000',
-                    'DTEND;TZID=\"(UTC-03:00) Brasília\":20170511T140000',
-                    'END:VEVENT',
-                    'END:VCALENDAR',
-                    ]
-
-        cal = icalendar.Calendar.from_ical('\r\n'.join(ical_str))
-        self.assertEqual(cal.errors, [])
```

Comment:
We are missing this assertion. Apart from that: I trust that you copied it alright and the other assertions see transferred.

Reply:
I readded the assertion in test_parses_event_with_non_ascii_tzid_issue_237